mlflow/gateway/client.py

```diff
@@ -117,13 +117,47 @@ def search_routes(self, search_filter: Optional[str] = None):
         response = self._call_endpoint("GET", self._route_base).json()["routes"]
         return [Route(**resp) for resp in response]
 
-    def query(self, route: str, data):
+    def query(self, route: str, data: Dict[str, Any]):
         """
         Submit a query to a configured provider route.
 
         :param route: The name of the route to submit the query to.
-        :param data: The data to send in the query. The format of the input data is route specific.
-        :return: The route's response as a dictionary
+        :param data: The data to send in the query. A dictionary representing the per-route
+        specific structure required for a given provider.
+
+        For chat, the structure should be:
+
+        .. code-block:: python
```

Comment:
```suggestion
        .. code-block:: python
        
```

We need a blank line here